src/config/env/test.js

```diff
@@ -1,7 +1,7 @@
 export default {
     env: 'test',
     db: 'mongodb://localhost/db',
-    port: 3000,
+    port: 3001,
```

Comment:
Server works on same (3000). I changed it to watch test and run server simultaneously.